tasks/discover-pdrs/schemas/config.json

```diff
@@ -37,6 +37,11 @@
       "default": false,
       "type": "boolean"
     },
+    "httpRequestTimeout": {
```

Comment:
Same comment I had elsewhere, it would be better to make it clear that this is a "crawler timeout" or "list timeout" or something.

Reply:
see: https://github.com/nasa/cumulus/pull/1755/files#r452447677

Reply:
e042a5278